p2p/discovery/message.go

```diff
@@ -196,14 +195,9 @@ func sendFindNodeRequest(u *udp, nodes []*Node, target NodeID) {
 		f := &findNode{
 			SelfID:  u.self.ID,
 			QueryID: target,
```

Comment:
I remember that, you'd better to add comment for these 3 members in definition to clarify their role.

Reply:
added